eng/automation/SignList.xml

```diff
@@ -32,7 +32,6 @@
     <ThirdParty Include="ShimSkiaSharp.dll" />
     <ThirdParty Include="Fizzler.dll" />
     <ThirdParty Include="Newtonsoft.Json.dll;" />
-    <ThirdParty Include="Svg2VectorDrawable.Net.dll" />
```

Comment:
Removing this svg2vector as the library is not currently used nor is it super useful if it is.